launcher/modManager/cmodlist.cpp

```diff
@@ -92,6 +140,15 @@ bool CModEntry::isUpdateable() const
 	return false;
 }
 
+bool CModEntry::isCompatible() const
+{
+	if(!isInstalled())
+		return false;
```

Comment:
Why false? Now it looks like if mod is not installed - it is not compatible which is a bit weird.

Reply:
it's because you can check compatibility for only installed mods.